tests/_support/AcceptanceTester.php

```diff
@@ -23,4 +23,39 @@ class AcceptanceTester extends \Codeception\Actor
    /**
     * Define custom actions here
     */
+
+   	/**
+	 * Sets tickets enabled post types
+	 *
+	 * @param  array $post_types The post types to be enabled for tickets. Can be empty.
+	 */
+	public function haveTicketablePostTypes ( array $post_types = array( ) ) {
+
+		$I = $this;
+
+		$tribe_options = $I->grabOptionFromDatabase( 'tribe_events_calendar_options' );
+		$tribe_options['ticket-enabled-post-types'] = $post_types;
+		$I->haveOptionInDatabase( 'tribe_events_calendar_options', $tribe_options );
+
+		/*$code = <<< PHP
+add_filter( 'tribe_tickets_post_types', 'test_ticketable_post_types' );
+function test_ticketable_post_types(){
+	return array( 'post' );
+}
+PHP;
+
+		$I->haveMuPlugin('ticketable-post-types.php',$code);*/
```

Comment:
any reason to keep these comments in place? if not, remove

Reply:
@lucatume I found the DB method slighly faster while running the tests but I'd defer to your opinion here.

Reply:
If they proved to be faster then use the method you found best and remove this code, thanks!

Reply:
Actually re-tested and it's not. I'll change back to the muplugin method.